### ISSUE TITLE
FR: Add e-commerce example that uses pop-over purchase sheet

### ISSUE THREAD

**Issue by ericlindley-g** [ampproject/amp-by-example#803]
Feature request to add an e-commerce pop-over sheet for completing a purchase (customizing) from a product detail page. [AliExpress](https://m.aliexpress.com/s/item/32798474850.html), for instance, uses this pattern, once you click "buy now".

Should be possible with amp-bind, but definitely worth flagging any barriers that need implementation in amphtml

/cc @sebastianbenz @kul3r4

**Comment by kul3r4** [comment]
Happy to take a look, which step of the purchase do you mean? "Review your order", "payment" or "done"? The page loads very very slow for me

**Comment by ericlindley-g** [comment]
(note that we can hold off on this until after I/O :)


If I open the link on mobile (I forgot to mention to use mobile—sorry about that!), I land here:

![screen shot 2017-05-11 at 8 15 50 am](https://cloud.githubusercontent.com/assets/16807766/25957020/2e6daa14-3622-11e7-8cca-c66cf738c91f.png)

When I click "buy now" a sheet pops up from the bottom:

![screen shot 2017-05-11 at 8 15 57 am](https://cloud.githubusercontent.com/assets/16807766/25957057/3e5bc2da-3622-11e7-8226-2d5414fa584e.png)

Which allows you to configure your purchase with additional options, without leaving the page.

The idea is to demo this general interaction of a popover sheet to configure the purchase with details not available on the main product page. I'm not sure what page has "review your order", "payment", or "done" on it — maybe that's on the desktop version?

**Comment by kul3r4** [comment]
Yes, that was on desktop. Sounds good, thanks for the snapshots!
